### PR TITLE
Allow manual build of the docker image

### PR DESCRIPTION
**Pull request by ppraegla**
This is the first part of the experiments in PR #17 . Once the `docker.yml` is on the default branch we can try to manually trigger the docker build and I will create another PR were we can try the docker update mechanism.

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -1,24 +1,67 @@
...
-      - 'dependencies/current/'
-      - 'dependencies/testing/'
+  workflow_dispatch:
+  # pull_request:
+  #   branches:
+  #     - 'main'
+  #   paths:
+  #     - '.github/workflows/docker.yml'
+  #     - 'dependencies/current/'
+  #     - 'dependencies/testing/'
```
Comment (amgebauer) [inline .github/workflows/docker.yml:11]:
```suggestion
```

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -1,24 +1,67 @@
...
 
+# TODO find a way to share these variables across the workflows
```
Comment (amgebauer) [inline .github/workflows/docker.yml:13]: I don't think there is a way currently.

```suggestion
```

**Review by amgebauer — COMMENTED** [review]
I cannot tell much about the remaining todos. What should we do with them?

File: .github/workflows/docker.yml (hunk 1 above)
Comment (ppraegla) [reply .github/workflows/docker.yml]: I removed the TODOs. The docker.yml needs a final overhaul anyway.

**Review by amgebauer — APPROVED** [review]

**Pull request merged**